metal3-dev/local-bmo.sh

```diff
@@ -15,9 +15,9 @@ if ! which yq 2>&1 >/dev/null ; then
     exit 1
 fi
 
-bmo_path=$GOPATH/src/github.com/metal3-io/baremetal-operator
-if [ ! -d $bmo_path ]; then
-    echo "Did not find $bmo_path" 1>&2
+BMO_PATH=${BAREMETAL_OPERATOR_PATH:-$GOPATH/src/github.com/metal3-io/baremetal-operator}
```

Comment:
For documentation purposes it could be useful to add it also in the `config_example.sh` file

Reply:
I did consider that, but it's not part of the regular `make` flow, so I figured it was more of an "advanced" case where folks will just look at the script?